coverage/.resultset.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
We should probably add coverage to the .gitignore so it wont keep track of the constant updating, but we can do that tomorrow when we set up the api service and facade!